setup.py

```diff
@@ -76,6 +76,9 @@
         "checklist==0.0.11",
         "wandb>=0.10.0,<0.12.0",
         "huggingface_hub>=0.0.8",
+        "datasets>=1.2.1",
```

Comment:
We should probably pin this tighter:
```suggestion
        "datasets>=1.2.1,<1.3.0",
```

Reply:
Why are we keeping it so small? Latest one is `1.10.2`, no?

Reply:
Oh, yea. So how about `<2.0`?

Reply:
done